extensions/wikia/Recirculation/RecirculationController.class.php

```diff
@@ -20,6 +21,7 @@ public function index() {
 		$posts = $fandomDataService->getPosts( $this->type );
 
 		if ( count( $posts ) > 0 ) {
+			$this->response->setCacheValidity( self::CACHE_DURATION );
```

Comment:
Yes `setCacheValidity` is dealing with Varnish/Browser cache. There is `WikiaResponse::CACHE_STANDARD`, but we decided on the 15 minute cache time to make sure we're grabbing fresh information from Parsely. Would it make sense to keep the Varnish cache for 24 hours even though we're updating the cache from Parsely every 15 minutes?